lib/engine/game/g_1860/step/dividend.rb

```diff
@@ -101,10 +101,6 @@ def payout_shares(entity, revenue)
             @log << "#{entity.name} pays out #{@game.format_currency(revenue)} = "\
                     "#{@game.format_currency(per_share)} (#{receivers})"
           end
-
-          def movement_str(times, dir)
-            "#{times / 2} #{dir}"
-          end
```

Comment:
Where did this go? I'm away from my dev machine so I can't tell where this was called.

Reply:
this is not longer needed, because the stepsize is now calculated correctly (doubling stepsize is not needed anymore)

Reply:
Nevermind. Found the call.

I'm curious what this will look like during the dividend step. Will it now show as "Right/Left" or "Up/Down"?

Reply:
Dividend steps use left right movement. Only stock selling or floating at the end of the SR use up/down. 

I also loaded a 62 and CZ and verified the view for dividend step looked correct.